pkg/kuttlctl/cmd/errors.go

```diff
@@ -0,0 +1,39 @@
+package cmd
+
+import (
+	"errors"
+
+	"github.com/spf13/cobra"
+
+	"github.com/kudobuilder/kuttl/pkg/test"
+)
+
+var (
+	errorsExample = `  # Asserts "errors" against a $KUBECONFIG cluster the values defined in the assert file.
+  kubectl kuttl errors <path/to/errorsfile.yaml>`
```

Comment:
Should this also accept multiple files for consistency with the `assert` command?

Reply:
it does... perhaps  you are looking for the example to show or reflect that... looking